jest.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  collectCoverage: true,
+  testMatch: ['**/*.spec.js'],
+  coverageDirectory: './coverage',
+  transform: {
+    '^.+\\.[jt]sx?$': require.resolve('babel-jest')
```

Comment:
Nit: might as well stay consistent with grove-core and use a simpler regex, since we don't use JSX or TypeScript.
```suggestion
    '^.+\\.js$': require.resolve('babel-jest')
```